src/test/scala/chiselTests/RecordSpec.scala

```diff
@@ -136,14 +136,14 @@ trait RecordSpecUtils {
     val k = new InnerInnerRecord
     val elements = SeqMap("" -> k)
     override def opaqueType = elements.size == 1
-    override def cloneType: this.type = (new SingleElementRecord).asInstanceOf[this.type]
+    override def cloneType: this.type = (new InnerRecord).asInstanceOf[this.type]
```

Comment:
@jackkoenig not sure how this didn't get caught on master..

Reply:
master does not call `.cloneType` as often (see https://github.com/chipsalliance/chisel3/pull/2611). Can you PR a fix to master as well?

Reply:
ah makes sense, yeah will do